packages/plugin-e2e/src/fixtures/commands/readProvisionedDashboard.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import { TestFixture } from '@playwright/test';
+import { promises } from 'fs';
+import { PluginFixture, PluginOptions } from '../../api';
+import { ReadProvisionedDashboardArgs } from '../../types';
+import { PlaywrightCombinedArgs } from '../types';
+
+type ReadProvisionedDashboardFixture = TestFixture<
+  <T = any>(args: ReadProvisionedDashboardArgs) => Promise<T>,
+  PluginFixture & PluginOptions & PlaywrightCombinedArgs
+>;
+
+const readProvisionedDashboard: ReadProvisionedDashboardFixture = async ({ provisioningRootDir }, use) => {
+  await use(async ({ fileName }) => {
+    const resolvedPath = path.resolve(path.join(provisioningRootDir, 'dashboards', fileName));
```

Comment:
**Just a nit**, not necessarily for this PR: maybe it would worth having these paths in a single place as constants?

Reply:
Yeah good shout - I'll do it in this PR (and for datasources too)